.github/tools/update_version.py

```diff
@@ -52,6 +52,7 @@ def get_hash(file):
         new_lines.append(new_version)
     else:
         new_lines.append(line)
+    new_lines.append("\n")
```

Comment:
This will create a newline for every single line making values.py something like this

```py
"""

Created by Epic at 9/1/20

Don't touch version, it is automatically generated.

"""

version = "0.1.59"

```
Please verify that your code works before making a PR